.goreleaser.yml

```diff
@@ -24,9 +25,7 @@ builds:
       - 7
 
 checksum:
-  # You can change the name of the checksums file.
-  # Default is `{{ .ProjectName }}_{{ .Version }}_checksums.txt`.
-  name_template: "{{ .ProjectName }}_{{ .Version }}_sha256-checksums.txt"
+  name_template: "{{ .ProjectName }}_{{ .Tag }}_sha256-checksums.txt"
```

Comment:
Could you just replace `sha256` with `sha512`? It's my mistake, I probably copied/pasted from 2 different sources 😬